tests/std/tests/LWG3987_including_flat_foo_doesnt_provide_begin_end/test.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <version>

```

Comment:
I think we should either avoid including any header, or include `<version>` and then guard tests with <code>_HAS_CXX*NN*</code>.